protocol/signalfx/signalfxforwarder.go

```diff
@@ -59,6 +62,7 @@ type ForwarderConfig struct {
 }
 
 var defaultForwarderConfig = &ForwarderConfig{
+	Filters:      filtering.FilterObj{}.Pointer(),
```

Comment:
It's just built ins you can't take literals to, like `&"hello"`.  You can do `&filtering.FilterObj{}`.